src/main/java/com/example/pfairplayservice/common/exception/EntityExceptionHandler.java

```diff
@@ -0,0 +1,47 @@
+package com.example.pfairplayservice.common.exception;
+
+import com.example.pfairplayservice.model.Member;
+import java.util.regex.Pattern;
+
+public class EntityExceptionHandler {
+
+    public static void MemberPostExceptionHandler(Member member) {
+
+        // null check
+        if (member.getId() == null) throw new RequiredParamNotFoundException("id를 입력해주세요.");
+        if (member.getPassword() == null) throw new RequiredParamNotFoundException("password를 입력해주세요.");
+        if (member.getName() == null) throw new RequiredParamNotFoundException("이름을 입력해주세요.");
+        if (member.getBirthday() == null) throw new RequiredParamNotFoundException("생년월일을 입력해주세요.");
+        if (member.getAddress() == null) throw new RequiredParamNotFoundException("주소를 입력해주세요.");
+        if (member.getPhoneNumber() == null) throw new RequiredParamNotFoundException("핸드폰번호를 입력해주세요.");
```

Comment:
The string can be an empty string. 
Use this code for String type. 
```java
if(StringUtils.isEmpty(member.getId())) 
    throw new ...
```

http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty-java.lang.CharSequence-

Reply:
I can't consider empty string. I will apply this.